paasta_tools/api/views/instance.py

```diff
@@ -881,8 +881,13 @@ def bounce_status(request):
         raise ApiFailure(
             "Temporary issue fetching bounce status. Please try again.", 599
         )
-    except Exception:
+    except Exception as e:
         error_message = traceback.format_exc()
+        if getattr(e, "status") == 404:
+            # some bounces delete the app & recreate
+            # in this case, we relay the 404 and cli handles gracefully
+            raise ApiFailure(error_message, 404)
```

Comment:
what does the error message look like again? if it's too low-level, maybe we hardcode in an error message like "Unable to find {instance} in Kubernetes - this is temporary for brutal bounces. If this error message persists, contact #compute-infra"?

Reply:
It'll be coming from here https://github.com/Yelp/paasta/blob/master/paasta_tools/cli/cmds/mark_for_deployment.py#L1652-L1661 and so is only controlling whether we return True/False for `check_if_instance_is_done` -- the error message still seems to make sense for this situation, although I think we'd written it expecting only brand-new deployments would hit it:
```
  "Can't get status for instance {}, service {} in "
                "cluster {}. This is normally because it is a new "
                "service that hasn't been deployed by PaaSTA yet.
```

Reply:
oh, that's perfect then - i'd say it's fine as-is :)